src/Config/MosaicConfigV2.ts

```diff
@@ -0,0 +1,152 @@
+import * as path from 'path';
```

Comment:
Now that #76 is merged, why does this appear as a whole new file? 🤔 Can you please rebase on latest `master` so we can see what's going on here?

Reply:
It doesn't show me as a new file. I have rearranged the sequence of classes to fix linter issues.